src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -44,6 +44,9 @@ const propTypes = {
         /** Is the user account validated? */
         validated: PropTypes.bool,
     }),
+
+    /* The workspace name */
+    policyName: PropTypes.string.isRequired,
```

Comment:
I don't think we need `policyName` to be required. The page still accessible without providing `policyID` to the route params.

Reply:
@mollfpr I just removed required and add the default policyName in all places. Could you help to review again